online-store/src/components/app/App.ts

```diff
@@ -0,0 +1,48 @@
+import { MODAL_WINDOW } from '../..';
+import { Product } from '../../domain/model';
+import { sourceData } from '../../domain/source';
+import { basketPageHTML } from './pages/basket';
+import { catalogPageHTML } from './pages/catalog';
+import { DetailsPage, updateProductInfo } from './pages/details';
+
+export class App {
+  contentContainer: HTMLDivElement | null;
+  constructor() {
+    this.contentContainer = document.querySelector('.root');
+  }
+
+  public renderMain() {
```

Comment:
`public` can be ommited, its default value :)

JFYI

Reply:
okei, thank you